pipeline_dp/pipeline_backend.py

```diff
@@ -190,11 +190,12 @@ def group_by_key(self, col, stage_name: str):
         """Group the values for each key in the PCollection into a single sequence.
 
         Args:
-          col: input collection
+          col: input collection (k, v)
           stage_name: name of the stage
 
         Returns:
-          An PCollection of tuples in which the type of the second item is list.
+          A PCollection of tuples in which the type of the second item is an
```

Comment:
Thanks for updating comments!

i.e. (key, Iterable[value])

**Explanation**: there is no `Iterable` interface, but there is [typing](https://docs.python.org/3/library/typing.html) annotations for Python. and typing.Iterable is annotations for iterable :), let's use it in this comment.